exp-player/addon/randomizers/pref-phys-pilot.js

```diff
@@ -344,12 +343,25 @@ function toFrames(frameId, eventVideos, BASE_DIR) {
                 BASE_DIR + 'stimuli/attention/',
                 'attentiongrabber'),
             sources: videoSourceObjs(
-		BASE_DIR + 'stimuli/' + features.eventType + '/',
-		e.fname, true),
+		        BASE_DIR + 'stimuli/' + features.eventType + '/',
+		        e.fname, true),
             altSources: videoSourceObjs(
                 BASE_DIR + 'stimuli/' + features.eventType + '/',
                 e.altName, true)
         };
+
+        // FOR PILOT ONLY: replace fall videos with calibration
+        if (e.compType === 'fall') {
+            returnFrame.sources = videoSourceObjs(
+                BASE_DIR + 'stimuli/attention/',
+                'calibration');
+            returnFrame.sources = videoSourceObjs(
```

Comment:
I've fixed it & can try it on the staging server tonight! (I did try the changes out locally, but forgot to try pausing one of the calibration videos. Thanks so much for the careful review!)

`e.compType === 'fall'` can happen on both the pilot and regular versions of the study, it's just that here we'll substitute in some calibration videos in place of the objects falling. For the actual study we care about kids' preferential looking during those videos; for the pilot, we just care about whether they find the videos overall reasonably engaging and whether we can code their direction of gaze. (The calibration videos are showing up correctly using this version of the prefphyspilot randomizer - every 4th video is a calibration video.)